CONSTITUTION.md

```diff
@@ -136,6 +137,10 @@ All members holding access privileges to the Society room shall make all reasona
 
 General Committee Officers are to be appointed at the discretion of the Executive Committee and removed at the earlier of either removal by discretion of the Executive Committee, or at the end of an office period of the Executive Committee. General Committee Officers do not hold a position on the Executive Committee, and shall assist and advise the Executive Committee. Persons eligible for appointments must be financial members of the Society for the year they are appointed. General Committee Officer roles and positions are to be determined and revised by the Executive Committee from time to time.
 
+### 5.8. Alumni Advisory Board
+
+Alumni Advisors are appointed upon the conclusion of their term on the Executive Committee or at the discretion of the Executive Committee, and hold a perpetual term until they choose to resign or are removed by the Executive Committee. Alumni Advisors do not hold a position on the Executive Committee, and shall serve a purely advisory role wherein, from time to time, they may provide considered guidance with no obligation on the Executive Committee. Persons considered for appointment must have held a role on the Executive Committee previously, or be recognised as a notable or otherwise distinguished member of the community.
```

Comment:
Assuming codification of unofficial long-standing practice is necessary:

1. Should a term be 'perpetual' or 'indefinite'? (which is the world I would use). The Executive Committee of the day should at least review the list annually if for no other reason than to be familiar with the alumni on the board.

2. What is "the community"?

Reply:
> Assuming codification of unofficial long-standing practice is necessary

We figured it would be worth debating at AGM.
Arguments For lean toward making future Execs keep the "Alumni" role around should the Discord be retired in favour of another platform, and that some form of direct contact is kept with past ProgSoc'ers (and therefore the history of the club).

> Should a term be 'perpetual' or 'indefinite'?

Wasn't entirely sure when I put assembled the proposal, you're probably right on using 'indefinite'. 

> review the list annually

I think this is a good argument against formalising current practice, because it would put some obligations on *all* previous ProgSoc'ers, instead of allowing for a passive connection should they choose.

> What is "the community"?

"The Club" is likely to be a clearer term. Though "member of the club" could imply current financial membership. Perhaps "ProgSoc Community" is a better middle ground.